src/components/AddHouseForm/AddHouseForm.tsx

```diff
@@ -24,6 +25,24 @@ import getRandomString from '/src/utils/getRandomString';
 import styles from './AddHouseForm.module.css';
 import FacilityCheckbox from './components/FacilityCheckbox/FacilityCheckbox';
 
+interface FieldsSchema {
```

Comment:
You have the interface `FormData` in the following PR and I think you could call this one here `FormData` as well.